pytorch_lightning/core/hooks.py

```diff
@@ -216,7 +216,7 @@ def on_validation_epoch_start(self) -> None:
         """
         # do something when the epoch starts
 
-    def on_validation_epoch_end(self, outputs: List[Any]) -> None:
+    def on_validation_epoch_end(self) -> None:
```

Comment:
@carmocca how do you recommend deprecating the `outputs` on `on_train_epoch_end` ? should it be typed as `Optional = None`

@kaushikb11 one of the assumptions here is that the trainer and lightning module signatures for these hooks will be the same, which isn't true in this case: https://github.com/PyTorchLightning/pytorch-lightning/blob/1c2ecbf70c38912124a384f273090145c1328353/pytorch_lightning/trainer/trainer.py#L1144-L1160

how do you recommend untangling this?